src/components/RoomView/Card.tsx

```diff
@@ -34,25 +33,18 @@ const Card = (word: CardObj) => {
 
   const submitAnswer = async (e: MouseEvent) => {
     e.preventDefault();
-    const target: string = (e.target as HTMLButtonElement).value;
-    let wordId = Number(target);
+    let wordId = word.id;
```

Comment:
this is cleaner / also had to do this because cards are now <ReactCardFlip> and not buttons so they cannot have a e.target.value anymore.